syntax = "proto3";

message GetUserRequest {
  int64 user_id = 1;
}

message User {
  string first_name = 1;
  optional string last_name = 2;
  optional string username = 3;
}

service TelegramClient {
  rpc GetUser(GetUserRequest) returns (User);
}